cmd/storagenode/main.go

```diff
@@ -266,6 +266,11 @@ func cmdSetup(cmd *cobra.Command, args []string) (err error) {
 	if !valid {
 		return fmt.Errorf("storagenode configuration already exists (%v)", setupDir)
 	}
+	
```

Comment:
CI is telling that this file isn't formatted accordingly gofmt and it's because of this line.